### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by bhaveshAn** [fossasia/susi_desktop#100]
**Actual Behaviour**

Currently, no CONTRIBUTING GUIDE is present in `.github` DIR which is required to follow best practices.

**Expected Behaviour**

CONTRIBUTING.md should be present in `.github` 

**Steps to reproduce it**

Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.

**LogCat for the issue**

Provide logs for the crash here

**Screenshots of the issue**

Where-ever possible attach a screenshot of the issue.

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
